frontend/vite.config.js

```diff
@@ -0,0 +1,15 @@
+import { defineConfig, loadEnv } from "vite";
+import solidPlugin from "vite-plugin-solid";
+
+export default ({ mode }) => {
+  process.env = { ...process.env, ...loadEnv(mode, process.cwd()) };
```

Comment:
Please, prove somehow that it is a good idea to extend the `process.env`

Reply:
in vite config we can use process env (without this setting we cant use it). In our code we can use only with import

https://stackoverflow.com/questions/75265156/how-to-add-environment-variables-in-vite-config-js-for-a-react-project